src/components/organisms/modal/VideoModal.tsx

```diff
@@ -1,32 +1,65 @@
-import { Card, CardMedia, Modal } from '@mui/material'
-import { useContext } from 'react'
+import { Card, CardMedia, Modal, SxProps } from '@mui/material'
+import { useContext, useEffect, useState } from 'react'
+import { ModalCloseButton } from '../../atoms/buttons/ModalCloseButton'
 import { Video } from '../../atoms/video/Video'
 import { ModalContext } from './ModalProvider'
-import { ModalCloseButton } from '../../atoms/buttons/ModalCloseButton'
+
+const sixteenByNineRatio = 16 / 9
+
+function getMoreImportantScreenValue(
+  height: number,
+  width: number
+): 'width' | 'height' {
+  return width / height > sixteenByNineRatio ? 'height' : 'width'
+}
 
 export function VideoModal() {
+  const { innerHeight: height, innerWidth: width } = window
+  const [, setWindowWidth] = useState(width)
```

Comment:
Assuming leaving the state variable out here was unintentional?
```suggestion
  const [windowWidth, setWindowWidth] = useState(width)
```

Reply:
Nope. Was intentional.  Many linters (I'm pretty sure Netlify is one of them) will not be happy if you declare unused variables.  We don't use `windowWidth` so Netlify will fail the build with that lint error.

One of the common ways to indicate that a variable is unused is to prefix it with a `_`  e.g.: `_windowWidth`.  I think this takes some amount of configuring on the linter side, hwoever.

The other way to indicate a variable is unused, is not to declare the variable at all (which is what we did here).

Reply:
That is what I thought was going on here. I have never actually seen it implemented so I aired on the side of caution (?) and flagged it for fixing. 

Appreciate the explanation and it makes perfect sense!